chrome-extension/js/background.js

```diff
@@ -154,11 +184,11 @@ function isEmpty(obj) {
 chrome.commands.onCommand.addListener((command) => {
   if (command == 'toggle-popup') {
     getCurrentTabUrl((url) => {
-      var key = getKey(url)
+      let key = getDomainKey(url);
```

Comment:
should we load the script on the frontend on load, and then check for the command pop-up on the client side? That way, we don't have to continuously message back n forth?
Can do for another PR

Reply:
Don't think so. I think the command pop-up listener should be the chrome extension's responsibility. Plus if we move this hook to the client, it looks like we'd have to move the bulk of the logic/fetching data to the client as well. Thoughts?